packages/generator/src/internal.ts

```diff
@@ -1,30 +1,29 @@
-export * from './operations';
 export * from './aggregator-package';
 export * from './batch';
 export * from './complex-type';
 export * from './edmx-parser';
 export * from './edmx-to-vdm';
 export * from './entity';
 export * from './enum-type';
-export * from './request-builder';
-export * from './sdk-metadata';
-export * from './service';
-export * from './swagger-parser';
 export * from './file-generator';
+export * from './generator';
 export * from './generator-options';
 export * from './generator-utils';
-export * from './generator';
+// Had to be added for public API check
```

Comment:
[pp] perhaps also add a comment that the `generator-without-ts-morph` will go away after complete migration.

Reply:
I don't remember touching this file, but it looks like this is done by the ide or linter.. anyway, I'll add a comment.